server/index.js

```diff
@@ -93,15 +104,18 @@ app.get('*', (req, res, next) => {
       title = title.substring(0, 47) + '...'
     }
 
-    description = description.split('"').join("'")
+    // Escape dynamic content before injecting into HTML
+    const safeTitle = escapeHtmlForAttribute(title)
+    const safeDescription = escapeHtmlForAttribute(description)
+    const safeOgUrl = escapeHtmlForAttribute('https://metriq.info' + req.url)
 
     // inject meta tags
     htmlData = htmlData
-      .replace(defaultTitle, title)
-      .replace(defaultTitle, title)
-      .replace(defaultDescription, description)
-      .replace(defaultDescription, description)
-      .replace("content='https://metriq.info'", "content='https://metriq.info" + req.url + "'")
+      .replace(defaultTitle, safeTitle)
+      .replace(defaultTitle, safeTitle)
+      .replace(defaultDescription, safeDescription)
+      .replace(defaultDescription, safeDescription)
```

Comment:
I know it was already that way before, but we could still take the chance to add a comment of why those calls are repeated twice (I am assuming title and descriptions appear in two tags)